### PR TITLE
Add an introduction to 1st_member_candidates.md

### PR DESCRIPTION
**Pull request by Zheng-Bicheng**
Add an introduction to 1st_member_candidates.md

### CODE REVIEW

**Comment by Zheng-Bicheng** [comment]
@Ligoml

File: pposdwg/1st_member_candidates.md [hunk 1]
```diff
@@ -64,10 +64,10 @@
 - 自我介绍：飞桨开发者技术专家（PPDE），飞桨仓库Committer，混迹于AiStudio和飞桨社区的各个角落，致力于创造有趣的项目。
 - 参加宣言：世界上最美好的一些事情，比如开源，总是真正自由并且富有创造力的！
 
-### 郑必城（GitHub ID：）
+### 郑必城（GitHub ID： [Zheng-Bicheng](https://github.com/Zheng-Bicheng)）
 
-- 自我介绍：
-- 参加宣言：
+- 自我介绍：PFCC、PPDE、Arm Ambassador，为Padlle2ONNX、FastDeploy、PaddlePaddle贡献过W行代码
```
Comment (Ligoml) [inline pposdwg/1st_member_candidates.md:69]: W行是多少行？

File: pposdwg/1st_member_candidates.md [hunk 2]
```diff
@@ -64,10 +64,10 @@
...
+- 参加宣言：不求最好, 但愿更好, 尽心尽力就是成功。
```
Comment (Ligoml) [inline pposdwg/1st_member_candidates.md:70]:
```suggestion
- 参加宣言：不求最好，但愿更好，尽心尽力就是成功。
```

**Comment by Zheng-Bicheng** [comment]
@Ligoml 再看一下👀

**Review by Ligoml — APPROVED** [review]

**Pull request merged**